public/index.html

```diff
@@ -2,6 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <meta property="og:image" content="shareable_image.png">
+    <meta property="og:image:type" content="image/png">
+    <meta property="og:image:width" content="1024">
+    <meta property="og:image:height" content="1024">
```

Comment:
This image height is incorrect. The `shareable_image.png` image is 628 pixels tall.